Classes/Issues/IssuesViewController.swift

```diff
@@ -352,6 +363,42 @@ IssueCommentSectionControllerDelegate {
             }
         }
     }
+    
+    func setLocked(_ locked: Bool) {
+        guard let currentStatus = current?.status else { return }
+        
+        let localModel = IssueStatusModel(
+            status: currentStatus.status,
+            pullRequest: currentStatus.pullRequest,
+            locked: locked
+        )
+        let localEvent = IssueStatusEventModel(
+            id: UUID().uuidString,
+            actor: client.sessionManager.focusedUserSession?.username ?? Strings.unknown,
```

Comment:
Can omit the `Strings` here I think

Reply:
Doesn't look like we can, the username is of type String but because the .unknown is namespaced into a Strings enum it can't be inferred

Reply:
Note to self, init this VC w/ the viewer or get name from GQL. Feels a little scary reaching into a session mgr for this and open/close status changes.

Reply:
Also I should come up w/ a way to better streamline creating local changes like this.

Reply:
Agree a better way for optimistic changes is needed, but it's worth noting that the name used there is only used locally for a short period of time. Soon as you reload the view via network it will update to be the correct name (if there was any confusion). This is purely to mock what will happen, and doesn't change what's sent via the API!

Reply:
Ya I know, I've just seen dependency on session managers leaking all over a codebase over time, turns into spaghetti code.